src/cast_core/test/SamplingLoader_test.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/json.hpp>
+
+#include <boost/exception/diagnostic_information.hpp>
+
+#include <yaml-cpp/yaml.h>
+
+#include <gennylib/context.hpp>
+#include <regex>
+#include <testlib/ActorHelper.hpp>
+#include <testlib/MongoTestFixture.hpp>
+#include <testlib/helpers.hpp>
+
+namespace genny {
+namespace {
+using namespace genny::testing;
+namespace bson_stream = bsoncxx::builder::stream;
+using namespace bsoncxx;
+
+TEST_CASE_METHOD(
+    MongoTestFixture,
+    "SamplingLoader - demo",
+    "[standalone][single_node_replset][three_node_replset][sharded][SamplingLoader]") {
```

Comment:
I noticed while authoring and copy/pasting from Loader_test.cpp, that it appears to have a typo or accidental '-1' in this 'single_node_replset' tag, which prevented me from testing this in that suite for a little while:
https://github.com/mongodb/genny/blob/46335eabb2d38cdeb5ea35122e29fa24d99ca53b/src/cast_core/test/Loader_test.cpp#L36

Is that a mistake worth fixing?